src/main/java/org/dependencytrack/integrations/defectdojo/DefectDojoUploader.java

```diff
@@ -65,6 +66,15 @@ public boolean isDoNotReactivateConfigured(final Project project) {
         }
     }
 
+    public boolean isVerifiedConfigured(final Project project) {
+        final ProjectProperty reimport = qm.getProjectProperty(project, DEFECTDOJO_ENABLED.getGroupName(), VERIFIED_PROPERTY);
+        if (reimport != null) {
+            return Boolean.parseBoolean(reimport.getPropertyValue());
+        } else {
+            return false;
```

Comment:
For backward compatibility, this should default to `true`, since the old behavior always set `verified` to `true`. Consider adding a comment here with this reason so future contributors know why it is like this.